profiles/serializers.py

```diff
@@ -88,24 +97,77 @@ class Meta:
 class UserWebsiteSerializer(serializers.ModelSerializer):
     """Serializer for UserWebsite"""
 
-    def validate_site_type(self, value):
+    def validate_url(self, value):
         """
-        Validator for site_type
-
-        NOTE: This is a no-op to allow for the calculation of site_type from the given url.
+        Validator for url. Prepends http protocol to the url if the protocol wasn't already included in the value.
         """
+        if not re.search(r"^http[s]?://", value):
```

Comment:
This should be case insensitive, if I enter 'HTTP://<something>' I end up with this:
<img width="325" alt="screen shot 2018-11-29 at 1 49 33 pm" src="https://user-images.githubusercontent.com/187676/49244492-ef206600-f3dd-11e8-965a-4ff1b984f36e.png">

Reply:
Still happening, this should fix it:
```python
re.search(r"^http[s]?://", value, re.IGNORECASE)
```